main.go

```diff
@@ -26,7 +26,7 @@ type Rsync struct {
 	Target    string   `json:"target"`
 	Delete    bool     `json:"delete"`
 	Recursive bool     `json:"recursive"`
-	Exclude   string   `json:"exclude"`
+	Exclude   []string `json:"exclude"`
```

Comment:
lets use `drone.StringSlice` type here. This allows the use of a string or an array in this field